models/articles.model.js

```diff
@@ -10,4 +10,16 @@ exports.getArticleById = (id) => {
         }
         return rows[0]
     })
-}
\ No newline at end of file
+}
+
+exports.updateArticleById = (id, increment) => {
+    return db.query('UPDATE articles SET votes = COALESCE(votes, 0) + $2 WHERE article_id = $1 RETURNING *', [id, increment]).then(({rows})=>{

```

Comment:
Coalesce here isn't really needed. If there are no votes on the patch request we don't need to make a database query.